etcdctl/ctlv3/ctl.go

```diff
@@ -124,6 +130,27 @@ func MustStart() {
 	}
 }
 
+func hideAllPersistentFlags() {
```

Comment:
nit, feel free to keep it as it's
```suggestion
func hideAllGlobalFlags() {
```